### PR TITLE
Organism populations

### PR DESCRIPTION
**Pull request by jomaora**
https://trello.com/c/XEpJIZOu/2380-organism-populations

**Detailed purpose of the PR**

Updating Chips atom and adding it to this new organism to be used to fetch, filter and select/unselect populations

**Result and observation**

Kapture 2021-11-16 at 15 17 51

**Testing Strategy**

- Already covered by tests
- Manual testing
- Unit testing

### CODE REVIEW

File: packages/@coorpacademy-components/src/atom/chips/index.js [hunk 1]
```diff
@@ -17,8 +17,10 @@ const Chips = props => {
       className={classnames(style.container, selected ? style.selected : style.unselected)}
       onClick={handleClick}
     >
-      <div className={style.text}>{text}</div>
-      <div className={style.information}>{information}</div>
+      <div className={style.textZone} title={text}>
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/atom/chips/index.js:20]: on ajoute l'aria-label ?

Comment (esa-coorp) [reply packages/@coorpacademy-components/src/atom/chips/index.js:20]: avec `text` et `information`?

Comment (esa-coorp) [reply packages/@coorpacademy-components/src/atom/chips/index.js:20]: vu que c'est un element cliquable

**Review by esa-coorp — CHANGES_REQUESTED** [review]

File: packages/@coorpacademy-components/src/atom/chips/index.js (hunk 1 above)
Comment (jomaora) [reply packages/@coorpacademy-components/src/atom/chips/index.js:21]: @esa-coorp done dans la div superior qui entoure tout le component (qui est clicable)

**Review by StefanoA1 — APPROVED** [review]

**Review by Fausto95 — APPROVED** [review]

**Pull request merged**